_commands/tcpdump.md

```diff
@@ -0,0 +1,101 @@
+22:59:52.973051 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [.], ack 110092589, win 65535, length 0
+22:59:52.973194 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [P.], seq 2068573471:2068573517, ack 110092589, win 65535, length 46
+22:59:52.973558 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [.], ack 2068573517, win 65535, length 0
+23:00:02.046795 IP 10.0.2.15.56854 > bud02s22-in-f196.1e100.net.http: Flags [.], ack 109761678, win 31844, length 0
+23:00:02.047116 IP bud02s22-in-f196.1e100.net.http > 10.0.2.15.56854: Flags [.], ack 1151493304, win 65535, length 0
+23:00:12.062722 IP 10.0.2.15.56854 > bud02s22-in-f196.1e100.net.http: Flags [.], ack 109761678, win 31844, length 0
+23:00:12.062908 IP bud02s22-in-f196.1e100.net.http > 10.0.2.15.56854: Flags [.], ack 1151493304, win 65535, length 0
+~~~
+##### Break it down
+-The -S flag prints absolute TCP sequence numbers versus relative
+-host <hostname> lets you specify a specific hostname to track requests and responses from a specific host (i.e. Google, Yahoo, eecs.umich.edu, etc.)
+#### Example command
+
+tcpdump -S "tcp[tcpflags] & (tcp-syn|tcp-ack|tcp-fin) != 0"
+
+~~~ bash
+$ sudo tcpdump -S "tcp[tcpflags] & (tcp-syn|tcp-ack|tcp-fin) != 0"
+tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
+listening on enp0s3, link-type EN10MB (Ethernet), capture size 262144 bytes
+23:13:35.006934 IP 10.0.2.15.44258 > ec2-204-236-230-155.compute-1.amazonaws.com.https: Flags [.], ack 210881678, win 32660, length 0
+23:13:35.007386 IP ec2-204-236-230-155.compute-1.amazonaws.com.https > 10.0.2.15.44258: Flags [.], ack 2435184464, win 65535, length 0
+23:13:38.226091 IP live.github.com.https > 10.0.2.15.47844: Flags [P.], seq 75075962:75075993, ack 2779464789, win 65535, length 31
+23:13:38.226285 IP 10.0.2.15.47844 > live.github.com.https: Flags [P.], seq 2779464789:2779464824, ack 75075993, win 39760, length 35
+23:13:38.226541 IP live.github.com.https > 10.0.2.15.47844: Flags [.], ack 2779464824, win 65535, length 0
+23:13:45.022884 IP 10.0.2.15.44258 > ec2-204-236-230-155.compute-1.amazonaws.com.https: Flags [.], ack 210881678, win 32660, length 0
+23:13:45.023070 IP ec2-204-236-230-155.compute-1.amazonaws.com.https > 10.0.2.15.44258: Flags [.], ack 2435184464, win 65535, length 0
+23:13:45.370676 IP 192.30.252.92.https > 10.0.2.15.41720: Flags [P.], seq 4164241:4164272, ack 185496046, win 65535, length 31
+23:13:45.370845 IP 10.0.2.15.41720 > 192.30.252.92.https: Flags [P.], seq 185496046:185496081, ack 4164272, win 39760, length 35
+23:13:45.371076 IP 192.30.252.92.https > 10.0.2.15.41720: Flags [.], ack 185496081, win 65535, length 0
+...
+... (continues)
+~~~
+##### Break it down
+-This command will let you watch the 3-way handshake between different clients and servers that are functioning on your network. Specifies that tcpflags are active and that at least one of the syn, ack, or fin flags are selected.
```

Comment:
This (and others) don't appear as a proper Markdown list. I think you need to put spaces in between your dash and your word.

- Hello there.